src/screens/Settings/SettingsPage.js

```diff
@@ -112,7 +111,7 @@ class SettingsPage extends Component {
             <Form>
               <Item>
                 <Label><Translate id="settings.language" /></Label>
-                <Text>{this.props.currentLanguage}</Text>
+                <Text>{this.props.activeLanguage.name}</Text>
```

Comment:
Where are we getting `activeLanguage` from?

Reply:
activeLanguage is a prop from react-localize-redux, along with the translate() function and languages object. These props are added to a component by wrapping the component in withLocalize().